MojioSDK/Models/User.swift

```diff
@@ -48,14 +48,13 @@ public class User : Object, Mappable {
         return self.Tags.toArray()
     }
 
-    public func json () -> String? {
+    public func jsonDict () -> NSDictionary {
```

Comment:
It was called json before, then I tried jsonObject. 

1. It was getting confusing with the toJSON() method of Alamofire
2. I would propose changing the RestClient so that it says json Dictionary rather than json object as I made the mistake of mixing up the two